src/main/perl/Lock.pm

```diff
@@ -1,11 +1,5 @@
-# ${license-info}
-# ${developer-info}
-# ${author-info}
-# ${build-info}
+#${PMpre} CAF::Lock${PMpost}
```

Comment:
yes, similar to the `${license-info}` etc
https://github.com/quattor/maven-tools/blob/master/build-profile/pom.xml#L25